exchanges/kraken/kraken_wrapper.go

```diff
@@ -357,24 +357,27 @@ func (k *Kraken) FetchTradablePairs(ctx context.Context, a asset.Item) (currency
 		}
 
 		pairs = make([]currency.Pair, 0, len(symbols))
-		for i := range symbols {
-			if strings.Contains(symbols[i].Altname, ".d") {
+		for _, info := range symbols {
+			if info.Status != "online" {
 				continue
 			}
-			base := assetTranslator.LookupAltname(symbols[i].Base)
+			if strings.Contains(info.Altname, ".d") {
+				continue
+			}
```

Comment:
Could you point me to why you've added this? It doesn't get hit, I don't see it in their documentation, nor was it in my old un-updated config

Reply:
I didn't it was already there. No idea, will remove.

Reply:
Previously Kraken use to add ".d" to the altname if a trading pair was disabled, before the status field was implemented